docs/syncer.md

```diff
@@ -11,64 +11,104 @@ users have to install a special component named [syncer](https://github.com/kcp-
 
 ## Instructions
 
-1. Create a workspace
+1. Create a workspace and immediately enter it:
 
 ```sh
-$ kubectl kcp workspace create my-workspace --use
-Workspace "my-workspace" created.
-Current workspace is "my-workspace".
+$ kubectl kcp workspace create my-workspace --enter
+Workspace "my-workspace" (type "Universal") created. Waiting for being ready.
+Current workspace is "root:default:my-workspace".
 ```
 
-Check that it has been created correctly and is ready:
+1. Obtain the server URL used to connect to KCP from its kubeconfig file:
 
 ```sh
-$ kubectl kcp workspace list
-NAME           TYPE        PHASE   URL
-my-workspace   Universal   Ready   https://192.168.1.150:6443/clusters/default:my-workspace--1
+$ export KCP_SERVER_URL=$(grep server .kcp/admin.kubeconfig | grep -v clusters | awk '{print $2}')
 ```
 
-Obtain the KCP server URL and the logical cluster name:
+1. Obtain the logical cluster name from `kubectl kcp workspace`:
 
-- KCP_SERVER_URL=https://192.168.1.150:6443
-- KCP_LOGICAL_CLUSTER_NAME=default:my-workspace--1
+```sh
+$ export KCP_LOGICAL_CLUSTER_NAME=$(kubectl kcp workspace | awk '{print $4}' | sed 's/".$/"/' | sed 's/"//g')
```

Comment:
Don't you need "current" here?
kubectl kcp workspace current | awk '{print $4}' | sed 's/".$/"/' | sed 's/"//g'

Reply:
Not any more